src/recon_buildblock/CMakeLists.txt

```diff
@@ -130,7 +130,7 @@ endif()
 
 # TODO what to do with IO?
 # modelling_buildblock currently needed for ParametricDensity and Patlak (TODO get rid of this somehow?)
-target_link_libraries(recon_buildblock modelling_buildblock display numerics_buildblock listmode_buildblock data_buildblock buildblock spatial_transformation_buildblock )
+target_link_libraries(recon_buildblock modelling_buildblock display numerics_buildblock listmode_buildblock data_buildblock buildblock spatial_transformation_buildblock local_motion_buildblock)
```

Comment:
We recently did a PR to some of the `Experimental` code, and we had a better way of adding libraries. Think we used `PARENT_SCOPE` from the Experimental `CMakeLists.txt`.

Reply:
I cannot quite recall what this was about, but it seems not so important at this point. we'd need to differentiate if `STIR_EXPERIMENTAL=ON` or not, unless we move motion things.

Reply:
You're right, not important, but thought the other way was better: https://github.com/UCL/STIR/pull/580/files#diff-5e304515000fc33e14a8d811e63d04d1R27-R30.

Reply:
seems different. this was creating a list of all registries/libraries. We cannot let `recon_buildblock` depend on `STIR_LIBRARIES` as it includes itself. We can also not let it depend on everything except itself, as then we create huge circles. I can't see how we can avoid doing this manually.

Probably better though would be to do
```cmake
if (BUILD_EXPERIMENTAL)
target_link_libraries(PoissonLogLikelihoodWithLinearKineticModelAndDynamicProjectionData.cxx.o local_motion_buildblock)
endif()
```
if such a thing is possible.